restore/data_test.go

```diff
@@ -21,6 +22,13 @@ import (
 )
 
 var _ = Describe("restore/data tests", func() {
+	var fpInfo filepath.FilePathInfo
+	copyFromPipeQueryTemplate := "COPY public.foo(i,j) FROM PROGRAM " +
+		"'(timeout 300 bash -c \"while [ ! -p \"%s\" ]; do sleep 1; done\" || (echo \"Pipe not found %s\">&2; exit 1)) && " +
+		"cat %s | cat - && test ! -e \"%s_error\"' WITH CSV DELIMITER ',' ON SEGMENT;"
```

Comment:
What's the purpose of `cat ... | cat -`? Doesn't this just output the same thing as the first command?

Reply:
It is done for uniformity of the COPY query (refer to `CopyTableIn()`).
In case the data is compressed, the second `cat` is replaced with the uncompression tool like `gzip` (or any other custom filter, returned by utils.GetPipeThroughProgram()).